static/js/login_or_signup.js

```diff
@@ -94,26 +106,49 @@ $(function() {
                     });
             }
         });
+    };
+
+    installFieldValidators(signupFieldsMetaData);
+    installFieldValidators(loginFieldsMetaData);
 
-    $signupForm.find('button').click(
+
+    var installSubmitClickHandler = function(fieldsMetaData, postFunction) {
```

Comment:
how do you feel about `install` vs `create` or any other synonym? `install` feels a bit weird to me

Reply:
please read response to previous comment.